### ISSUE TITLE
Adjust.onCreate on coroutineScope.launch(ioDispatcher)

### ISSUE THREAD

**Issue by ravikant42** [adjust/android_sdk#621]
I have the scenario in which I wish to call the onCreate on the background thread. onResume will be called on main thread.
Actually we are reading the tracker value from different source which may take some time and want to call the Adjust.onCreate only after that.
Will this cause any issue?

**Comment by uerceg** [comment]
Hey @ravikant42,

This shouldn't be any issue at all. Even if invoked from the main thread, SDK is processing any call made to it in the background thread, so it shouldn't really matter which thread you are making the call from.

Of course, if for your particular case you do notice something not working as expected, please let us know, but you shouldn't face any issues.

**Comment by shashanksu** [comment]
Hi @ravikant42 ,

Thank you for reaching out. Since we haven't received any further responses, we are closing this issue for now. If there's more information to provide, please feel free to reopen it or create a new issue. We appreciate your understanding and support!
